client/state/themes/actions.js

```diff
@@ -355,14 +361,22 @@ export function requestActiveTheme( siteId ) {
  * Triggers a network request to activate a specific theme on a given site.
  * If it's a Jetpack site, installs the theme prior to activation if it isn't already.
  *
- * @param  {string}   themeId   Theme ID
- * @param  {number}   siteId    Site ID
- * @param  {string}   source    The source that is reuquesting theme activation, e.g. 'showcase'
- * @param  {boolean}  purchased Whether the theme has been purchased prior to activation
- * @returns {Function}           Action thunk
+ * @param  {string}   themeId      Theme ID
+ * @param  {number}   siteId       Site ID
+ * @param  {string}   source       The source that is reuquesting theme activation, e.g. 'showcase'
+ * @param  {boolean}  purchased    Whether the theme has been purchased prior to activation
+ * @returns {Function}             Action thunk
  */
 export function activate( themeId, siteId, source = 'unknown', purchased = false ) {
 	return ( dispatch, getState ) => {
+		// Show auto loading homepage warning?
```

Comment:
Minor: is there anything documenting what this warning is actually all about? I feel like we could do with something which explains it or refers to the GH Issue or something.